theme/static/css/hydroshare_extends.css

```diff
@@ -306,4 +306,17 @@ float the buttons inside it. This prevents collapsing. */
     overflow-y: auto;
     /* prevent horizontal scrollbar */
     overflow-x: hidden;
```

Comment:
I'm interested in the history of css/hydroshare_extends.css and css/hydroshare_core.css can you share anything you know about the differences or is usage of one or the other fairly arbitrary?

Reply:
I only know that those css files are written mostly by Maurier.